tensorflow/tools/ci_build/xla/linux/rocm/run_py3.sh

```diff
@@ -47,6 +47,7 @@ bazel test \
       --build_tests_only \
       --test_output=errors \
       --test_sharding_strategy=disabled \
+      --test_env=TF2_BEHAVIOR=0 \
```

Comment:
This will force all tests in the testsuite to run in r1.x mode.  That is probably not what we want to do.

It may make more sense to have the script run two sets of tests for the time being
1. the six tests that need TF2_BEHAVIOR=0
2. the rest of the tests

The env var should be set for the first set of tests only

see the CSB script for an example of a script running two sets of tests ( https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/blob/develop-upstream/tensorflow/tools/ci_build/linux/rocm/run_csb_tests.sh )

Reply:
@deven-amd Thus far there is no evidence to support XLA unit tests can run in V2 mode so I chose to force use 1.x mode.

Reply:
@deven-amd i just changed the test script to cope with your requests.